knowledge_repo/converters/html.py

```diff
@@ -34,6 +34,31 @@
                         'knowledge_repo.converters.html:IndentsAsCellOutput']
 
 
+class IndentsAsCellOutputPreprocessor(Preprocessor):
+    """
+    Ensure all indented blocks are followed by a blank line to allow html
+    preprocessors to extract html elements (like scripts) properly.
+    """
+
+    def run(self, lines):
+        in_block = False
+        block_startable = True
+        for i, line in enumerate(lines):
+            if not line.startswith(' ' * self.markdown.tab_length):
+                if in_block:
+                    if line != "":
+                        lines.insert(i, "")
+                    in_block = False
+                else:
+                    if line == "":
+                        block_startable = True
+                    else:
+                        block_startable = False
```

Comment:
That's true. Would make it easier to read too :). Will fix before merging. Thanks Nikki.